src/main/java/com/google/cloud/tools/login/GoogleLoginState.java

```diff
@@ -380,11 +381,22 @@ public void initialize(HttpRequest httpRequest) throws IOException {
   private void persistCredentials() {
     Preconditions.checkState(isLoggedIn());
 
+    List<Exception> errors = new ArrayList<>();
     for (Account account : accountRoster.getAccounts()) {
       OAuthData oAuthData = new OAuthData(account.getAccessToken(), account.getRefreshToken(),
           account.getEmail(), account.getName(), account.getAvatarUrl(),
           oAuthScopes, account.getAccessTokenExpiryTime());
-      authDataStore.saveOAuthData(oAuthData);
+      try {
+        authDataStore.saveOAuthData(oAuthData);
+      } catch (Exception ex) {
+        loggerFacade.logError("Exception saving authorization data for " + account.getEmail(), //$NON-NLS-1$
```

Comment:
May not matter right now, but account email is the sort of PII that makes it hard to auto-collect logs.

Reply:
Good point.